titus_isolate/isolate/workload_manager.py

```diff
@@ -70,9 +95,21 @@ def __remove_workload(self, workload_id):
         if workload_id not in self.__workloads:
             raise ValueError("Attempted to remove unknown workload: '{}'".format(workload_id))
 
-        self.__cpu_allocator.free_threads(workload_id)
-
-        self.cpu = self.__cpu_allocator.get_cpu()
+        allocator = self.__cpu_allocator
+        try: 
```

Comment:
Is this copy pasted from above?  Can be factored out?

Reply:
Done